apps/src/templates/teacherDashboard/AddInitialStudentsView.jsx

```diff
@@ -0,0 +1,146 @@
+import React, {Component, PropTypes} from 'react';
+import i18n from '@cdo/locale';
+import color from '../../util/color';
+import experiments from '../../util/experiments';
+import CardContainer from './CardContainer';
+import LoginTypeCard from './LoginTypeCard';
+
+const style = {
+  h1: {
```

Comment:
Definite candidates to hoist into a separate .js file as discussed elsewhere.  Whoever does it first gets their work used by a lot of components.  color.js seems like a reasonable example to follow (though it happens to be generated by .scss apparently).